### ISSUE TITLE
kbd macro doesn't work

### ISSUE THREAD

**Issue by marcdexet-cnrs** [asciidoctor/asciidoctor-reveal.js#276]
Hi.

I'm' using asciidoctor 2.0.0

I'm using same source for html5, pdf and revealsjs for courses with a few adaptations depending the backend.

But I discoverd that kbd macro doesn't work with the revealjs backend.
It is not interpreted.
Do I miss something ?

**Comment by Mogztter** [comment]
Hi @marcdexet-cnrs 

I believe the `kbd` macro is correctly converted to HTML using the following template:

https://github.com/asciidoctor/asciidoctor-reveal.js/blob/45a02329209d5a48a2ba6932102376189f35c222/templates/inline_kbd.html.slim#L1-L8

But I think there's no style associated with `span.keyseq`.
To workaround this issue, you can provide a custom stylesheet with the following rules:

```css
.keyseq{color:rgba(51,51,51,.8)}
kbd{font-family:"Droid Sans Mono","DejaVu Sans Mono",monospace;display:inline-block;color:rgba(0,0,0,.8);font-size:.65em;line-height:1.45;background:#f7f7f7;border:1px solid #ccc;-webkit-border-radius:3px;border-radius:3px;-webkit-box-shadow:0 1px 0 rgba(0,0,0,.2),0 0 0 .1em white inset;box-shadow:0 1px 0 rgba(0,0,0,.2),0 0 0 .1em #fff inset;margin:0 .15em;padding:.2em .5em;vertical-align:middle;position:relative;top:-.1em;white-space:nowrap}
.keyseq kbd:first-child{margin-left:0}
.keyseq kbd:last-child{margin-right:0}
```

Taken from: https://github.com/asciidoctor/asciidoctor/blob/master/data/stylesheets/asciidoctor-default.css

@obilodeau Do you want to add built-in support? ie. add the above CSS rules in https://github.com/asciidoctor/asciidoctor-reveal.js/blob/master/templates/asciidoctor_revealjs.css.slim ?

**Comment by obilodeau** [comment]
FYI #77 tracks compliance issues like this one.

**Comment by obilodeau** [comment]
Support for this was just merged and will be part of the upcoming 4.0.0 release.
